src/main/java/karen/command/InvalidCommand.java

```diff
@@ -0,0 +1,53 @@
+package karen.command;
+
+import karen.KarenException;
+import karen.Storage;
+import karen.Ui;
+
+/**
+ * To indicate invalid user inputs.
+ */
+public class InvalidCommand extends Command {
+    public static final String DEFAULT_MESSAGE = "I don't understand anything - I want to speak with your manager";
+    private final String message;
+
+    /**
+     * Constructor function to override default message of InvalidCommand
+     * @param message custom message to describe invalidity
+     */
+    public InvalidCommand(String message) {
+        this.message = message;
```

Comment:
The usage of this keyword 'this' is fine as the parameter message shadows the object variable message. 👍